src/tests/ftest/util/dmg_utils.py

```diff
@@ -915,6 +917,40 @@ def pool_evict(self, pool, sys=None):
         """
         return self._get_result(("pool", "evict"), pool=pool, sys=sys)
 
+    def config_generate(self, access_points=None, num_pmem=None, num_nvme=None,
+                        net_class=None):
+        """Produce a server configuration.
+
+        Args:
+            access_points (str): Comma separated list of access point addresses.
+                Defaults to None.
+            num_pmem (int): Minimum number of SCM (pmem) devices required per
```

Comment:
this actually specifies the exact number of pmem devices to use rather than a minimum, I may need to update dmg manpage for that

Reply:
Will change.